README.md

```diff
@@ -1,27 +1,38 @@
 # obsgit
+
 Simple bridge between Open Build Server and git.
 
 This tools can be used to export a project stored in OBS into a local
 git repository, and later imported from git to the same (or different)
 OBS server.
 
+## Instalation
```

Comment:
Maybe a nit, but IMHO this and all the rest should be first level.

Reply:
Well, going with some standards, one is h1 level, usually the title, everything else should be h2, but I can change if you prefer.